tilavarauspalvelu/settings.py

```diff
@@ -518,3 +519,23 @@ def get_git_revision_hash() -> str:
     # NOTE: If you run this in docker,
     # here is additional information for set-up: https://django-debug-toolbar.readthedocs.io/en/latest/installation.html
     # right before "Troubleshooting"
+
+# GraphQL debug toolbar and django debug toolbar cannot be enabled at the same time
+USE_GRAPHQL_DEBUG_TOOLBAR = env("USE_GRAPHQL_DEBUG_TOOLBAR")
```

Comment:
I think these should be defined in local_settings and then this information how to conf these would be in README

Reply:
Hmm, I think you have a good point here! These affect performance significantly, and should only be used in development, so, I agree it would be a good idea to move configuration into the documentation. I'll remove the ENV variables and move this to documentation.

Is it too much of a hassle to install packages so that the debug packages would not be included in the requirements list?

Reply:
I don't think it's too much of a hassle. In future we could have separate requirements for dev as we discussed irl while back.